pom.xml

```diff
@@ -199,6 +199,13 @@
             <version>1.0.0</version>
         </dependency>
 
+        <!-- Apicurio Registry native client -->
+        <dependency>
+            <groupId>io.apicurio</groupId>
+            <artifactId>apicurio-registry-client</artifactId>
+            <version>2.2.3.Final</version>
+        </dependency>
```

Comment:
This dependency should be marked as **optional**. 

Users that doesn't use apicurio-registry doesn't need this jar and users who do will just declare the dependency again in their project.